sysboundary/sysboundary.cpp

```diff
@@ -388,7 +392,9 @@ bool SysBoundary::classifyCells(dccrg::Dccrg<spatial_cell::SpatialCell,dccrg::Ca
    for (int x = 0; x < localSize[0]; ++x) {
       for (int y = 0; y < localSize[1]; ++y) {
          for (int z = 0; z < localSize[2]; ++z) {
-            technicalGrid.get(x,y,z)->sysBoundaryFlag = sysboundarytype::NOT_SYSBOUNDARY;
+            //technicalGrid.get(x,y,z)->sysBoundaryFlag = sysboundarytype::NOT_SYSBOUNDARY;
+            // Here for debugging since boundarytype should be fed from MPIGrid
+            technicalGrid.get(x,y,z)->sysBoundaryFlag = sysboundarytype::N_SYSBOUNDARY_CONDITIONS;
```

Comment:
Is this N_SYSBOUNDARY_CONDITIONS (basically a guaranteed illegal value) on purpose, so that the simulation crashes when updating from vlasov grid fails later on?

Reply:
That's how I read it (or rather, *if* it fails to get it from MPIGrid then we do see it with this value.)

Reply:
Yes. It's initialized to invalid value here, and later _should_ get the correct value from the MPI grid.